refactor(docs): Refine Kudu connector page (#27961)
prestodb/presto: presto-docs/src/main/sphinx/connector/kudu.rst

``etc/catalog/kudu.properties`` with the following contents,
replacing the properties as appropriate:

.. code-block:: properties

    connector.name=kudu

    ## List of Kudu master addresses, at least one is needed (comma separated)
    ## Supported formats: example.com, example.com:7051, 192.0.2.1, 192.0.2.1:7051,
    ##                    [2001:db8::1], [2001:db8::1]:7051, 2001:db8::1
    kudu.client.master-addresses=localhost

    ## Kudu does not support schemas, but the connector can emulate them optionally.
    ## By default, this feature is disabled, and all tables belong to the default schema.
    ## For more details see connector documentation.
    #kudu.schema-emulation.enabled=false

    ## Prefix to use for schema emulation (only relevant if `kudu.schema-emulation.enabled=true`)
    ## The standard prefix is `presto::`. Empty prefix is also supported.
    ## For more details see connector documentation.
    #kudu.schema-emulation.prefix=

    #######################
    ### Advanced Kudu Java client configuration
    #######################

>     ## Default timeout used for administrative operations (e.g. createTable, deleteTable, etc.)

Reviewer: ```suggestion
    ## Default timeout used for administrative operations such as createTable or deleteTable.
```
Avoid Latin abbreviations. See the [GitLab documentation style guide recommended word list entry for e.g.](https://docs.gitlab.com/development/documentation/styleguide/word_list/#eg) for discussion and alternatives.

> E.g. To query a Kudu table named ``special.table!`` use ``SELECT * FROM kudu.default."special.table!"``.

Reviewer: ```suggestion

To query a Kudu table named ``special.table!`` use ``SELECT * FROM kudu.default."special.table!"``.
```
